price/priceService.ts

```diff
@@ -0,0 +1,34 @@
+import { baseUrl } from './url.js';
+import {Pair} from '../types/common.js'
+
+type TimeFrame = string;
+
+interface Candle {
```

Comment:
This type will be reused. So it makes sense to move it to common types file instead of keeping it here.  
Same applies to `TimeFrame`.

Reply:
ok sure

Reply:
The interface is still in the service file.

Reply:
ok sure.
but in common.ts there is a candle type . can I remove it ?
export type Candle = any;